packages/examples/src/example.ts

```diff
@@ -22,25 +22,31 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   THE SOFTWARE.
 */
-import { JsonSchema, UISchemaElement } from '@jsonforms/core';
+import { 
+  JsonFormsCellRendererRegistryEntry,
+  JsonFormsUISchemaRegistryEntry,
+  JsonFormsRendererRegistryEntry,
+  JsonSchema,
+  UISchemaElement
+} from '@jsonforms/core';
 
 export interface ExampleDescription {
   name: string;
   label: string;
   data: any;
   schema: JsonSchema;
   uischema: UISchemaElement;
+  uischemas?: JsonFormsUISchemaRegistryEntry[];
   config?: any;
+  actions?: any;
```

Comment:
I think we can add these types here.
```suggestion
  actions?: {label: string, apply: (props: StateProps) => any}[];
```